light_curves/code/fluxconversions.py

```diff
@@ -23,22 +24,55 @@ def convert_WISEtoJanskies(mag, magerr, band):
     flux uncertaintiy: array
         uncertainty on the returned flux corresponding to input magerr
     """
-    if band == 'w1':
-        zpt = 309.54
-    elif band == 'w2':
-        zpt = 171.787
-            
-    flux_Jy = zpt*(10**(-mag/2.5))
-    
-    #calculate the error
-    magupper = mag + magerr
-    maglower = mag - magerr
-    flux_upper = abs(flux_Jy - (zpt*(10**(-magupper/2.5))))
-    flux_lower = abs(flux_Jy - (zpt*(10**(-maglower/2.5))))
+    # wise_df["flux"] = convert_wise_flux_to_millijansky(nanomaggy_flux=wise_df["flux"])
+    # wise_df["dflux"] = convert_wise_flux_to_millijansky(nanomaggy_flux=wise_df["dflux"])
```

Comment:
Photometry is stored in the catalog as fluxes "in Vega nanomaggies". So, it seems like we should be able to do a direct unit conversion like these commented-out lines of code (function defined below). But, this is not returning the same fluxes as the original code. I suspect this is related to zero points.

The original code received magnitudes from unTimelyCatalogExplorer and converted them flux, so the code below does the same transformations (flux -> magnitude -> flux) in order to produce the same results as the original.

Reply:
@jkrick will you take a look at these flux conversions? If this is confusing, I'm happy to talk through it on a call.

To try to be a little more clear, right now this code is returning the same fluxes as the original code. If that is correct, there is no need to change anything. It seems unnecessary to go flux -> magnitude -> flux, but it doesn't cause any problems (compute time is negligible). I just want someone else to look at it because I'm getting confused about why the direct unit conversion doesn't work, and whether/why different zero points are used in the flux -> magnitude vs the magnitude -> flux.